src/main/java/nextstep/subway/domain/Section.java

```diff
@@ -0,0 +1,116 @@
+package nextstep.subway.domain;
+
+import javax.persistence.*;
+
+@Entity
+public class Section extends BaseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "up_station_id", foreignKey = @ForeignKey(name = "fk_section_to_up_station"))
+    private Station upStation;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "down_station_id", foreignKey = @ForeignKey(name = "fk_section_to_down_station"))
+    private Station downStation;
+
+    @Column(name = "distance")
```

Comment:
```suggestion
    @Column
```

변수명과 동일하다면 name을 따로 지정하지 않아도 됩니다 ㅎㅎ